servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/single/ReduceSingleTest.java

```diff
@@ -128,22 +124,18 @@ public void subscribeOnOriginalIsPreserved() throws Exception {
         from("Hello").beforeRequest(__ -> {
             if (currentThread() == testThread) {
                 errors.add(new AssertionError("Invalid thread invoked request-n. Thread: " +
-                        currentThread()));
+                                              currentThread()));
             }
             analyzed.countDown();
-        }).subscribeOn(executorRule.executor()).collect(ArrayList::new, (objects, s) -> {
+        }).subscribeOn(executorExtension.executor()).collect(ArrayList::new, (objects, s) -> {
             objects.add(s);
             return objects;
         }).toFuture().get();
         analyzed.await();
         assertThat("Unexpected errors observed: " + errors, errors, hasSize(0));
     }
 
-    private static class ReducerRule extends Verifier {
```

Comment:
@Scottmitch I do not see the need for `ReducerRule`. It does not have any state and does not override `Verifier.verify` callback. Replaced it with a simple method call.